packages/salesforcedx-utils-vscode/test/unit/helpers/traceFlags.test.ts

```diff
@@ -32,10 +31,11 @@ describe('Trace Flags', () => {
   let toolingUpdateStub: SinonStub;
 
   beforeEach(async () => {
-    sb = createSandbox();
+    sb = $$.SANDBOX;
```

Comment:
what does this do? :)

Reply:
Instead of creating our own sandbox we are using the pre-built sandbox from `TestContext()`